src/components/ClusterNodes/NodeActions/EditTaintsModal.tsx

```diff
@@ -0,0 +1,230 @@
+                                        </span>
+                                    )}
+                                </div>
+
+                                <div className="w-70 mr-8">
+                                    <ReactSelect
+                                        options={TAINT_OPTIONS}
+                                        onChange={(selectedValue) => {
+                                            onEffectChange(selectedValue, index)
+                                        }}
+                                        data-index={index}
+                                        components={{
+                                            IndicatorSeparator: null,
+                                            DropdownIndicator,
+                                            Option: (props) => (
+                                                <Option
+                                                    {...props}
+                                                    tippyClass="default-tt w-200"
+                                                    showTippy={true}
+                                                    placement="left"
+                                                    tippyContent={props.data['description']}
+                                                />
+                                            ),
+                                        }}
+                                        value={{
+                                            label: taintDetails.effect,
+                                            value: taintDetails.effect,
+                                        }}
+                                        styles={{
+                                            ...containerImageSelectStyles,
+                                            singleValue: (base, state) => ({
+                                                ...base,
+                                                padding: '5px 0',
+                                            }),
+                                        }}
+                                    />
+                                </div>
+                                <div>
+                                    <DeleteIcon
+                                        className="icon-dim-20 mt-4 pointer"
+                                        data-index={index}
+                                        onClick={deleteTaint}
+                                    />
+                                </div>
+                            </div>
+                        )
+                    })}
+                </div>
+                <div className="dc__border-top flex right p-16">
+                    <button className="cta cancel h-36 lh-36 mr-12" type="button" onClick={onClose}>
+                        {EDIT_TAINTS_MODAL_MESSAGING.Actions.cancel}
+                    </button>
+                    <button className="cta h-36 lh-36" onClick={onSave}>
```

Comment:
disable both buttons when `apiCallInProgress` is true

Reply:
Done